content/blog/2022/09/2022-09-18-digital-ocean-sponsorship.adoc

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Digital Ocean and Jenkins partnership continues to grow!"
+tags:
+- jenkins
+- digitalocean
+- community
+author: kmartens27
+description: "DigitalOcean increases Jenkins sponsorship"
+opengraph:
+  image: /images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png
+---
+
+We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
```

Comment:
Optional suggestion. 
```suggestion
We are excited to announce that the relationship between DigitalOcean and Jenkins continues to grow!
```

Reply:
same idea,  much better and smoother to read, thanks so much @kellie-freeman !!